spid_cie_oidc/onboarding/validators/authn_response.py

```diff
@@ -0,0 +1,21 @@
+
+from pydantic import BaseModel, constr
+from typing import Literal
+
+class AuthenticationResponse(BaseModel):
+    # TODO: is 1*VSCHAR
+    code: str
+    state: constr(min_length = 32)
+
+class AuthenticationResponseCie(AuthenticationResponse):
+    iss: Literal['https://idserver.servizicie.interno.gov.it']
```

Comment:
please move the constant `'https://idserver.servizicie.interno.gov.it'` to test settings.py file and import it

in otherwords, don't disseminate the same copy/paste of code

Reply:
I have defined a constant ISS = Literal['https://idserver.servizicie.interno.gov.it'] in settings.
First I defined ISS = 'https://idserver.servizicie.interno.gov.it' but in this way I received this error message:
Type arguments for "Literal" must be None, a literal value (int, bool, str, or bytes), or an enum valuePylance

Reply:
Please use only a Httpurl without any literal. A rp must be tested on a demo op

Reply:
So have I only to test that it is an Httpurl and not its value?

Reply:
yes, it's needed for demo propose

the demo OP will be on http://localhost:8002

this is why we need to have the CIE url in the global settings, in this way the project would be configured for every needs without touching the code.

the formula we adopt for this kind of feature is this
https://github.com/peppelinux/spid-cie-oidc-django/blob/261082bcc248d5252c1361acf7091778223706a7/spid_cie_oidc/entity/jwtse.py#L15

it means get this PARAMETER in the global settings and if it's not available take it from local/default settings